packages/react-app/src/hooks/useLocalState.js

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useEffect, useState } from 'react';
+
+const useLocalState = (
+  initialValue,
+  key = 'omnibridge-key',
+  valueType = 'string',
+) => {
+  const storageValue = localStorage.getItem(key);
+  let castedValue = storageValue;
+
+  if (valueType === 'number') {
+    castedValue = parseInt(storageValue, 10);
+  } else if (valueType === 'boolean') {
+    castedValue = Boolean(storageValue);
+  } else if (valueType === 'object') {
+    castedValue = JSON.parse(storageValue);
+  }
+
+  const [value, setValue] = useState(castedValue || initialValue);
+
+  const updateValue = useCallback(
+    val => {
+      console.log(val, typeof val);
```

Comment:
Can we remove this unnecessary console.log?
Would be useful to run `yarn lint --fix` before pushing :)